static/mobile.css

```diff
@@ -0,0 +1,56 @@
+@media only screen and (max-device-width: 750px) {
+    /* Make header links visible */
+    body > header > .row {
+        height: auto;
+    }
+
+    body > header > .row > .hide-on-print {
+        background-color: #AD6A91;
+        border-radius: 4px;
+    }
+
+    /* Add margin between main content hero boxes since they will be on top
+     * of one another */
+    .hero > .col#first {
+        margin-bottom: 2em;
+    }
+
+    /* Add margin between download / doc buttons since they will be on top
+     * of one another */
+    .hero > .col#second a.primary {
+        margin-bottom: 1em;
+    }
+
+    /* Anchor footer to bottom, decrease font size
+     * NB: padding-bottom and height will need to be adjusted in the future
+     *     if more content is added to the footer! */
+    html {
+        position: relative;
+    }
+
+    body, html {
+        min-height: 100%;
+        margin: 0;
+    }
+
+    body > #content {
+        padding-bottom: 5em;
+        margin-bottom: 0;
+    }
+
+    body > footer {
+        position: absolute;
+        bottom: 0;
+        height: 5em;
+        font-size: 10pt;
+    }
```

Comment:
Totally agree on this. Checkout [the demo page](http://g7sweb.uwtb.net) now and see if that's better (clearing cache if necessary).